babel.jest.config.js

```diff
@@ -2,5 +2,6 @@ module.exports = {
   presets: [
     ["@babel/preset-env", { targets: { node: "current" } }],
     ["@babel/preset-react", { runtime: "automatic" }],
+    '@babel/preset-typescript',
```

Comment:
This repo's `jest.config.js` configured Jest to use Babel to transpile modules with this `babel.jest.config.js` file. `@babel/preset-typescript` is the fast+recommended way to get Babel transpiling TypeScript.

https://babeljs.io/docs/en/babel-preset-typescript